### ISSUE TITLE
add doc

### ISSUE THREAD

**Issue by rom1504** [LAION-AI/CLIP-based-NSFW-Detector#1]
1. methodology for training set collection
2. methodology for test set collection
3. command to run the training
